rust/datafusion/src/datasource/datasource.rs

```diff
@@ -24,6 +24,15 @@ use crate::arrow::datatypes::SchemaRef;
 use crate::error::Result;
 use crate::physical_plan::ExecutionPlan;
 
+/// The table statistics
```

Comment:
I suggest clarifying in the comment if the statistics are meant to be a hint or accurate. Specifically, it would help to know if other parts of the system should rely on them being correct or if they are simply a hit.

Maybe @andygrove  has some thoughts as he was the one who filed https://issues.apache.org/jira/browse/ARROW-10781

Reply:
I agree. These should be considerd estimates IMO.

Reply:
Thanks @alamb I have fixed the code.